webview-ui/src/components/settings/providers/Unbound.tsx

```diff
@@ -47,6 +141,28 @@ export const Unbound = ({ apiConfiguration, setApiConfigurationField, routerMode
 					{t("settings:providers.getUnboundApiKey")}
 				</VSCodeButtonLink>
 			)}
+			<div className="flex justify-end">
+				<Button variant="outline" onClick={handleRefresh} className="w-1/2 max-w-xs">
+					<div className="flex items-center gap-2 justify-center">
+						<span className="codicon codicon-refresh" />
+						{t("settings:providers.refreshModels.label")}
+					</div>
+				</Button>
+			</div>
+			{didRefetch && (
+				<div className="flex items-center text-vscode-charts-green">
+					{t("settings:providers.refreshModels.success", {
+						defaultValue: "Models list updated! You can now select from the latest models.",
+					})}
+				</div>
+			)}
+			{isInvalidKey && (
+				<div className="flex items-center text-vscode-errorForeground">
+					{t("settings:providers.invalidApiKey", {
+						defaultValue: "Invalid API key. Please check your API key and try again.",
```

Comment:
Can you use Translate mode to add translations for these strings in all supported languages? We prefer to just translate into all and not have a defaultValue. Happy to help, let me know.